app/services/indexes_attributes.rb

```diff
@@ -66,11 +66,8 @@ def authority_label(obj)
     end
 
     def label_for_resource_type(cv_list, uri)
-      cv_list.each do |pair|
-        if pair[1] == uri
-          return pair[0]
-        end
-      end
-      uri
+      label = cv_list.find {|pair| pair[1] == uri }
```

Comment:
Looks good. This could, I think, also be made into a one-liner with a con of potentially being a bit harder to read. With that in mind, here's that one-liner approach that you can consider. Overall, I'm 👍 on this PR.

As for the one-liner, I believe you can pass a proc to `find` in the case where there isn't a match. So, you might be able to do something like:
```
Array(cv_list.find( proc { [uri] } ) { |pair| pair[1] == uri }).first
```